### ISSUE TITLE
[RN] Unable to get the package working...

### ISSUE THREAD

**Issue by itseramin** [lucaburgio/iconoir#54]
## Prerequisites

* Version: 4.3.1
* Are you running from source/master: y
* Are you using a released build: probably
* Operating system: macOS 11.4
* Bits:

## Step to reproduce

Step 1: Use React Native
Step 2: Try to use the package:
```
import { Iconoir } from 'iconoir-react-native';
/* ... */
<Iconoir color={'black'} height={25} width={25} />
```
Step 3: Fail miserably :(

### Actual behavior

_see screenshot below..._

## Any message or error

_see screenshot below..._

## Additional info or screenshots

I'm not sure if it's the devDependencies, because it looked like they were not installed, so I manually installed them, but I still can't get the package working... Also my project is free of Typescript code maybe that could be the root of all of this?

<img width="734" alt="Screenshot 2021-06-20 at 1 57 43" src="https://user-images.githubusercontent.com/55285477/122658267-76d04a80-d16b-11eb-8747-6a16b44275b5.png">
<img width="381" alt="Screenshot 2021-06-20 at 3 32 05" src="https://user-images.githubusercontent.com/55285477/122659397-26131e80-d178-11eb-81d2-92e8e48f95ea.png">

**Comment by lucaburgio** [comment]
@paescuj @PianoManDanDan

**Comment by paescuj** [comment]
I took a quick glance but couldn't find the cause.
Unfortunately, I've no time to take a closer at the moment, but will do asap.

**Comment by itseramin** [comment]
@paescuj I really appreciate your work bringing this package to React Native! Thanks in advance!

**Comment by paescuj** [comment]
We should not include 'react-native-svg' in our bundle - it should be installed separately (like react & react-native).
I've published [iconoir-react-native@2.0.1](https://www.npmjs.com/package/iconoir-react-native) where this has been fixed.
Please see #56 for new installation instruction.
Tested with expo!
